python-client/pyproject.toml

```diff
@@ -88,7 +88,7 @@ include = ["giskard*"]
 exclude = ["docs*", "tests*"]
 
 [tool.setuptools.package-data]
-giskard = ["**/*.html", "**/*.css", "**/*.js", "**/*.json", "**/*titanic.csv"]
+giskard = ["**/*.html", "**/*.css", "**/*.js", "**/*.json", "**/*titanic.csv","**/*prompts.csv","**/*occupations.txt","**/*template.txt"]
```

Comment:
Maybe `**/*.txt` is ok. I don’t see cases where this would include unwanted things.